package.json

```diff
@@ -10,8 +10,8 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "test": "jest --roots src --watch",
-    "test:ci": "jest --roots src",
+    "cy": "cypress open",
+    "test:ci": "cypress run --component",
```

Comment:
cy:ci ça serait mieux non ?

Reply:
Justement je sais pas si on bouge sur sur 
```
 "cy": "cypress open",
 "cy:ci": "cypress run --component",
```
ou si on garde 

```
  "test": "cypress open",
  "test:ci": "cypress run --component",
```

Mais oui clairement avoir le melange n'est pas logique

Reply:
I would stay with `test` as it is conventional. We also might change `cypress` for something else in the future.